types/amap-js-api/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for non-npm package amap-js-sdk 1.4
```

Comment:
the package name should be amap-js-api, right?

Reply:
thank for your review, yes, this is my mistake,i will fix it